sd_jwt/operations.py

```diff
@@ -231,7 +233,7 @@ def _create_signed_jwt(self):
 
         # Sign the SD-JWT using the issuer's key
         self.sd_jwt = JWS(payload=dumps(self.sd_jwt_payload))
-        _headers = {"alg": self._sign_alg, "kid": self._issuer_key.thumbprint()}
+        _headers = {"alg": self._sign_alg}
```

Comment:
@bc-pi I removed the "kid" header parameter from the examples. It used to contain the key thumbprint. AFAIK there is no requirement for a "kid" header parameter and just always putting the thumbprint there is not the right solution anyway - any objections?

Reply:
(I'm not sure why `kid` was in there in the first place.)

Reply:
It was me @danielfett, it's not relevant for the draft and its examples, please go ahead :-)

Reply:
no objections